nixos/modules/services/misc/home-assistant.nix

```diff
@@ -245,22 +245,84 @@ in {
         rm -f "${cfg.configDir}/ui-lovelace.yaml"
         ln -s ${lovelaceConfigFile} "${cfg.configDir}/ui-lovelace.yaml"
       '');
-      serviceConfig = {
-        ExecStart = "${package}/bin/hass --config '${cfg.configDir}'";
+      serviceConfig = let
+        # List of capabilities to equip home-assistant with, depending on configured components
+        capabilities = [
+          # Empty string first, so we will never accidentally have an empty capability bounding set
+          ""
```

Comment:
Went with an empty string as recommended in https://github.com/NixOS/nixpkgs/issues/120617#issuecomment-830685115.